pyproject.toml

```diff
@@ -13,17 +13,16 @@ authors = [{ name = "Meadsteve", email = "steve@meadsteve.dev" }]
 license = { file = "LICENSE" }
 classifiers = [
   "License :: OSI Approved :: MIT License",
-  "Programming Language :: Python :: 3.7",
-  "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
   "Topic :: Software Development :: Libraries",
   "Typing :: Typed",
 ]
 dependencies = []
-requires-python = ">=3.7"
+requires-python = ">=3.9"
```

Comment:
Can we keep this on 3.7. Although it's not formally supported it *should* work so I'd like to have it installable for any user willing to do test coverage themselves.